noir/aztec_macros/src/lib.rs

```diff
@@ -313,12 +335,24 @@ fn check_for_aztec_dependency(
     crate_id: &CrateId,
     context: &HirContext,
 ) -> Result<(), (MacroError, FileId)> {
+    if has_aztec_dependency(crate_id, context) {
+        Ok(())
+    } else {
+        let crate_graph = &context.crate_graph[crate_id];
+        Err((AztecMacroError::AztecDepNotFound.into(), crate_graph.root_file_id))
+    }
+}
+
+fn has_aztec_dependency(
+    crate_id: &CrateId,
+    context: &HirContext,
+) -> bool {
     let crate_graph = &context.crate_graph[crate_id];
     let has_aztec_dependency = crate_graph.dependencies.iter().any(|dep| dep.as_name() == "aztec");
     if has_aztec_dependency {
-        Ok(())
+        true
     } else {
-        Err((AztecMacroError::AztecDepNotFound.into(), crate_graph.root_file_id))
+        false
     }
```

Comment:
You can simply replace this entire thing for the expr `crate_graph.dependencies...`, skipping the intermediate variable, if/else etc